src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,86 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import cn from 'classnames';
+import { SearchLink } from './SearchLink';
+import { SelectedSex } from '../types/SelectedSex';
+
+type PeopleFiltersProps = {
+  query: string,
+  sex: string,
+  centuries: string[],
+  handleSex: (newSex: string) => void,
+  handleQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void,
+};
+
+export const PeopleFilters: React.FC<PeopleFiltersProps> = ({
+  query,
+  sex,
+  centuries,
+  handleSex,
+  handleQueryChange,
+}) => {
+  const allCentury = ['16', '17', '18', '19', '20'];
+
+  function toggleCentury(century: string) {
+    return centuries.includes(century)
+      ? centuries.filter(num => num !== century)
+      : [...centuries, century];
+  }
+
+  const renderCenturyButtons = () => {
```

Comment:
I advise you to watch this video:
https://www.youtube.com/watch?v=vsIeKWSOQSM&ab_channel=%D0%9A%D1%83%D1%80%D1%81React%D0%B4%D0%BB%D1%8F%D0%BF%D0%BE%D1%87%D0%B0%D1%82%D0%BA%D1%96%D0%B2%D1%86%D1%96%D0%B2%D0%B2%D1%96%D0%B4VitaliiRuban

Reply:
If nothing has changed, I understand that you did not watch the video 😢